### DISCUSSION TITLE
2D Game example

### DISCUSSION THREAD

**Discussion by aliessmael** [mono/SkiaSharp#2160] [Ideas]
Thanks so much for this library.
I am wondering can we have simple 2D game example, I don't want to use engine for that just very simple game.

**Comment by mattleibow** [comment]
@bijington just messaged me last week with this: https://github.com/bijington/orbit

**Reply by bijington** [reply]
Thanks for the shoutout. And what great timing 😄

**Comment by aliessmael** [comment]
Thanks so much, exactly what I am looking for.
